clab/netlink.go

```diff
@@ -61,12 +58,12 @@ func (c *cLab) createVethWiring(l *Link) error {
 	}
 
 	// create veth pair in the root netns
-	linkA, linkB, err := CreateVethIface(ARndmName, BRndmName, l.MTU)
+	linkA, linkB, err := createVethIface(ARndmName, BRndmName, l.MTU)
 	if err != nil {
 		return err
 	}
-	vA.Link = &linkA
-	vB.Link = &linkB
+	vA.Link = linkA
+	vB.Link = linkB
```

Comment:
what I meant here, is to do this instead:
```
var err error 
vA.Link, vB.Link, err = createVethIface(ARndmName, BRndmName, l.MTU)
```
no need to allocate the vars linkA and linkB

Reply:
done in 583d1ec